src/org/traccar/processing/CopyAttributesHandler.java

```diff
@@ -32,9 +32,9 @@ private Position getLastPosition(long deviceId) {
     @Override
     protected Position handlePosition(Position position) {
         String attributesString = Context.getDeviceManager().lookupAttributeString(
-                position.getDeviceId(), "processing.copyAttributes", null, true);
+                position.getDeviceId(), "processing.copyAttributes", Position.KEY_DRIVER_UNIQUE_ID + ",", true);
```

Comment:
But split will still work. It will return single parameter, as we want it to. In your case it will return 2 string with empty string as a second one.